Xamarin.Forms.Core/Shapes/EllipseGeometry.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Xamarin.Forms.Shapes
+{
+    public class EllipseGeometry : Geometry
+    {
+        public static readonly BindableProperty CenterProperty =
+            BindableProperty.Create(nameof(Center), typeof(Point), typeof(EllipseGeometry), new Point());
+
+        public static readonly BindableProperty RadiusXProperty =
+            BindableProperty.Create(nameof(RadiusX), typeof(double), typeof(EllipseGeometry), 0.0);
+
+        public static readonly BindableProperty RadiusYProperty =
+            BindableProperty.Create(nameof(RadiusY), typeof(double), typeof(EllipseGeometry), 0.0);
```

Comment:
I prefer the Geometry API over the Shape ones. can't we unify ?

Reply:
Yes, we can review and unify. Right now the Spec is based on the WPF API (allow porting code, etc). For this reason, Line has X1, X2, Y1, Y2 properties: https://docs.microsoft.com/en-us/dotnet/api/system.windows.shapes.line?view=netcore-3.1 while LineGeometry has two points: https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.linegeometry?view=netcore-3.1
In this case, personally I like more to have the two points.